src/main/java/org/apache/commons/configuration2/DatabaseConfiguration.java

```diff
@@ -478,6 +478,21 @@ protected Boolean performOperation() throws SQLException {
         return result != null && result.booleanValue();
     }
 
+    @Override
+    protected boolean containsValueInternal(final String value) {
+        final AbstractJdbcOperation<Boolean> op = new AbstractJdbcOperation<Boolean>(ConfigurationErrorEvent.READ, ConfigurationErrorEvent.READ, value, null) {
+            @Override
+            protected Boolean performOperation() throws SQLException {
+                try (ResultSet rs = openResultSet(String.format(SQL_GET_PROPERTY, table, valueColumn), false, value)) {
+                    return rs.next();
+                }
+            }
+        };
+
```

Comment:
No need for extra whitespace (blank lines). If you want to call out a code fragment, then you can use an inline `//` comment to alert the reader to something specific.

Reply:
Corrected.